server/app/routes.py

```diff
@@ -14,12 +15,20 @@ def hello_world():
 		return 'Hello World!'
 
 	@app.route('/videos_raw')
-	def videos_raw():
-		return jsonify(rss.raw_feed(app.config['watchlist']))
+	async def videos_raw():
+		feed = await asyncio.to_thread(lambda: rss.raw_feed(app.config['watchlist']))
```

Comment:
what's the upside of doing this if we are still executing one thing at a time and waiting for it to finish? this kinda stuff usually makes sense if you are launching several operations in parallel (e.g. launch fetching of each channel in a separate thread).

Reply:
The main thread that listens for requests will be able to handle those requests while the fetch is happening in the background.

Not a big difference since there is only one intended client, but now it won't block if frontend were to make multiple requests at a time in the future.

I don't think this would have any benefit if we were to use a production WSGI server in the future. Since those should
come with parallelism.

Reply:
A better solution would be to make the rss module asynchronous, since parallel fetches could be implemented there, but that isn't on my priority list as of now.